### ISSUE TITLE
AWS authentication requires a valid Date or x-amz-date header

### ISSUE THREAD

**Issue by mavencode01** [hubrick/vertx-s3-client#5]
First of all, thank you for this library.

I ran into this exception:
```
AWS authentication requires a valid Date or x-amz-date header
```

am I doing anything wrong?

**Comment by marcust** [comment]
Apparently, but most likely we do.
What exactly are you doing, got some example code?

**Comment by opyate** [comment]
Same here.
Response:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<Error><Code>AccessDenied</Code><Message>AWS authentication requires a valid Date or x-amz-date header</Message><RequestId>1337BE4A420769DF</RequestId><HostId>1337k+cL0ypAnnj2T3T9k1ohjl0521wr2Rw/jhE+5W2KPvI+PTHTPymWk4ITeUhw/Gx5gYZ6BCQ=</HostId></Error>
```

I'm just using the PUT example as per the README, but using my own credentials.

**Comment by opyate** [comment]
Actually, that `20170509T154625+01` looks like you didn't test the library during British Summer Time ;-)
I think Amazon wants UTC. I'll do a little experiment to verify.

**Comment by opyate** [comment]
@marcust please see the PR. I look forward to this being published on Maven Central :)

**Comment by marcust** [comment]
Merged and released as 2.1.1 ... thanks for your input.
